.gitattributes

```diff
@@ -0,0 +1,13 @@
+.editorconfig          export-ignore
+.gitattributes         export-ignore
+/.github/              export-ignore
+.gitignore             export-ignore
+/*.md                  export-ignore
+/LICENSE.md            -export-ignore
+/README.md            -export-ignore
```

Comment:
Not sure about how you'd like to indent these, but it looks like it's off by 1 space on either one of these lines :)